hnclient-v2/src/main/java/ca/bc/gov/hlth/hnclientv2/Route.java

```diff
@@ -75,8 +77,12 @@ public void configure() throws Exception {
         retrieveAccessToken = new RetrieveAccessToken(tokenEndpoint, scopes, clientAuthBuilder);
         // TODO this might be better to just be run from main but requires a property loader and modifying the retrieveAccessToken
         renewKeys();
+        
+        onException(org.apache.http.conn.HttpHostConnectException.class).process(new FailureProcessor())
```

Comment:
It might be out of scope of this story but do we need another exception processor to handle other exceptions? Like ones thrown by the HandshakeServer, the util class, the RetrieveAccessToken class, etc.